src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1406,9 +1416,8 @@
   (slurp cacrl))
 
 (schema/defn ^:always-validate
-  get-crl-last-modified :- DateTime
-  "Given the value of the 'cacrl' setting from Puppet, return
-  a Joda DateTime instance of when the CRL file was last modified."
+  get-file-last-modified :- DateTime
+  "Given the a path to a file return a Joda DateTime instance of when the CRL file was last modified."
```

Comment:
Also may be? "Given path to a file, return a Joda DateTime instance of when the file was last modified."